services/auth-query-service/AuthQueryService.Api/.env.example

```
@@ -0,0 +1,65 @@
+# MongoDB Configuration (Read Database)
+# ================================================
+# Connection string format: mongodb://username:password@host:port
+# For local development without auth:
+#   MONGODB_CONNECTION_STRING=mongodb://localhost:27017
+# With authentication:
+#   MONGODB_CONNECTION_STRING=mongodb://admin:password@localhost:27017
+
+MONGODB_CONNECTION_STRING=mongodb://admin:123@localhost:27017/?authSource=admin
+MONGODB_DATABASE=auth_query
+
+Mongo__ConnectionString=mongodb://admin:123@localhost:27017/?authSource=admin
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The Mongo__ConnectionString key should be in uppercase

(LowercaseKey)